scripts/release-notes.py

```diff
@@ -43,6 +43,20 @@ def num_commits_since_prev_tag(token, base_url):
     print(f"There are {num_commits} new commits since {prev_release_tag}")
     return num_commits
 
+categories = {
+        '#### ⛔ Breaking Changes': 'changelog:breaking-change',
+        '#### New Features': 'changelog:new-feature',
+        '#### Bug fixes, Minor Improvements': 'changelog:bugfix-or-minor-feature',
+        '#### 🚧 Experimental Features': 'changelog:exprimental',
+        '#### CI Improvements': 'changelog:ci',
```

Comment:
Do you want to add `test` and `skip` labels? We should auto-exclude them from the generated list

Reply:
yeah, we can have them too. making a change 🚀